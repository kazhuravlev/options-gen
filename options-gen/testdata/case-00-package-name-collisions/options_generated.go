// Code generated by options-gen qa-version. DO NOT EDIT.

package testcase

import "net/http"

type OptOptionsSetter func(o *Options)

func NewOptions(
	F1 http.Client,
	options ...OptOptionsSetter,
) Options {
	var o Options

	// Setting defaults from field tag (if present)

	o.F1 = F1

	for _, opt := range options {
		opt(&o)
	}
	return o
}

func (o *Options) Validate() error {
	return nil
}
