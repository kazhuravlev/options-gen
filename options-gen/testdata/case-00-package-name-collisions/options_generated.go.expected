// Code generated by options-gen qa-version. DO NOT EDIT.

package testcase

import http "github.com/kazhuravlev/options-gen/options-gen/testdata/case-00-package-name-collisions/pkg/net/api"

type OptOptionsSetter func(o *Options)

func NewOptions(
	F1 http.Client,
	options ...OptOptionsSetter,
) Options {
	var o Options

	// Setting defaults from field tag (if present)

	o.F1 = F1

	for _, opt := range options {
		opt(&o)
	}
	return o
}

func (o *Options) Validate() error {
	return nil
}
